README.md

```diff
@@ -0,0 +1,47 @@
+<div align="center">
+
+[![Gaggiuino](/images/GAGGIUINO_LOGO_transp.png)](https://gaggiuino.github.io/#/)
+
+[![Discord Chat](https://img.shields.io/discord/890339612441063494)](https://discord.gg/eJTDJA3xfh "Join Discord Help Chat")
+</div>
+
+# Gaggiuino Documentation
+
+This repo holds the documentation for the [Gaggiuino] project. It uses [docsify] to render static pages via [GitHub Pages].
+
+## Local Development and Testing
+
+### Using Docsify
+
+First, install docsify globally:
+
+```
+npm i docsify-cli -g
+```
+
+Run a local web server on the `./docs` folder:
+
+```
+docsify serve docs
+```
+
+
+### Using Python
```

Comment:
Why even have this when all of this is achieved via the previous step ? Plus doing it via docsify ensures the rendering is going to be the same.

Reply:
I had it in there because it's a nice alternative to using Node.js to install docsify. If for whatever reason, someone can't use Node or can't install Node, they can still use Python to preview the site. This Python method is referenced in the docsify documentation as well:

https://docsify.js.org/#/quickstart?id=manually-preview-your-site

I'll submit another PR clarifying this.